flink-container/src/main/java/org/apache/flink/container/entrypoint/jobgraph/StandaloneJobGraphClusterConfiguration.java

```diff
@@ -0,0 +1,55 @@
+package org.apache.flink.container.entrypoint.jobgraph;
+
+import org.apache.flink.api.common.JobID;
+import org.apache.flink.runtime.entrypoint.EntrypointClusterConfiguration;
+import org.apache.flink.runtime.jobgraph.SavepointRestoreSettings;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+
+import java.util.Properties;
+
+import static org.apache.flink.util.Preconditions.checkNotNull;
+
+/**
+ * Configuration for the {@link StandaloneJobGraphClusterEntryPoint}.
+ */
+public class StandaloneJobGraphClusterConfiguration extends EntrypointClusterConfiguration {
```

Comment:
One nit comment and a question:
* nit: Generally, I think it makes sense to have `@Nullable` arguments last if possible instead of as intermediate args.
* ~Since we are the only users of this entry point, I would suggest to make JobID required as well since we expect to always set this in order to have a 1:1 mapping between VVP job ID and Flink job ID. What do you think?~ We actually already set the ID [during translation](https://github.com/dataArtisans/vvp-services/blob/9f2481fa06b46ab7cce965e208ee04edde06ef57/vvp-sql-service/src/main/java/org/apache/flink/container/entrypoint/JobGraphTranslator.java#L212-L217), making the job ID argument obsolete.